api/api.go

```diff
@@ -107,6 +95,18 @@ func (w Wrapper) RegisterEndpoint(ctx echo.Context) error {
 	return ctx.NoContent(http.StatusCreated)
 }
 
+func (w Wrapper) DeleteEndpoint(ctx echo.Context, idStr string) error {
+	id, err := ssi.ParseURI(idStr)
+	if err != nil {
+		return echo.NewHTTPError(http.StatusInternalServerError, fmt.Errorf("invalid endpoint id: %w", err))
+	}
+
+	if err := w.SPService.DeleteEndpoint(*id); err != nil {
+		return echo.NewHTTPError(http.StatusInternalServerError, err.Error())
+	}
+	return ctx.NoContent(200)
```

Comment:
I'd say HTTP 204 No Content instead of 200 (with 200 some response body is expected)